Source/Core/VideoCommon/BPStructs.cpp

```diff
@@ -753,61 +753,808 @@ void LoadBPRegPreprocess(u32 value0)
   }
 }
 
-void GetBPRegInfo(const u8* data, std::string* name, std::string* desc)
+template <>
+struct fmt::formatter<GenMode>
 {
-  const char* no_yes[2] = {"No", "Yes"};
+  constexpr auto parse(format_parse_context& ctx) { return ctx.begin(); }
+  template <typename FormatContext>
+  auto format(const GenMode& mode, FormatContext& ctx)
+  {
+    const char* cullmodes[] = {"None", "Back-facing primitives only",
```

Comment:
```suggestion
    static constexpr std::array cullmodes = {"None", "Back-facing primitives only",
```

Ditto elsewhere where applicable. Otherwise MSVC generally constructs the arrays on the stack every time the function is executed, as opposed to just storing the data in the R/O segment.